client/src/main/java/tw/waterball/survive/io/client/gameloop/GameLoop.java

```diff
@@ -1,14 +1,20 @@
 package tw.waterball.survive.io.client.gameloop;
 
-import tw.waterball.survive.io.client.controller.EventController;
+import tw.waterball.survive.io.client.sprite.Sprite;
 import tw.waterball.survive.io.client.view.GameView;
 
+import java.util.List;
+
 public class GameLoop {
-	private GameView gameView;
-	private EventController eventController;
+    private GameView gameView;
+    private List<Sprite> sprites;
 
-	public void start() {
+    public GameLoop(GameView gameView, List<Sprite> sprites) {
```

Comment:
Don't barely pass the `List` in, you must encapsulate the sprites with some classes, for example a `SpriteContainer` class